packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -325,10 +255,9 @@ function Session(props: PropsWithChildren<Props>) {
           tdpClient.nuke();
         }}
         userHost={`${username}@${hostname}`}
-        clipboardSharingEnabled={clipboardSharingActive}
         canShareDirectory={directorySharingState.canShare}
         isSharingDirectory={directorySharingState.isSharing}
-        onShareDirectory={onShareDirectory}
+        {...props}
```

Comment:
`TopBar` isn't actually using any of these props that are being passed in from `DesktopSession` so I think this can be removed.

Reply:
It's using `clipboardSharingEnabled, onShareDirectory, warnings, onRemoveWarning`